clients/src/main/java/org/apache/kafka/clients/consumer/Consumer.java

```diff
@@ -95,8 +95,14 @@
     /**
      * @see KafkaConsumer#commitSync(Map)
      */
+    @Deprecated
     void commitSync(Map<TopicPartition, OffsetAndMetadata> offsets);
 
+    /**
+     * @see KafkaConsumer#commitSync(Map, Duration
+     */
+    public void commitSync(final Map<TopicPartition, OffsetAndMetadata> offsets, 
```

Comment:
nit: the `public` is redundant for an interface. Same for the other methods below.